prometheus/desc.go

```diff
@@ -127,7 +126,7 @@ func NewDesc(fqName, help string, variableLabels []string, constLabels Labels) *
 		labelNameSet[labelName] = struct{}{}
 	}
 	if len(labelNames) != len(labelNameSet) {
-		d.err = errors.New("duplicate label names")
+		d.err = fmt.Errorf("duplicate label names in constant and variable labels for metric %q", fqName)
```

Comment:
Should we also print `labelNames`? (:

Reply:
I wasn't sure, I was trying to walk the line between specific enough to give you a place to debug, without giving a flood of unstructured logging. If you think printing the full array is the right move, I'm happy to include that.

Reply:
IMO it's useful, usually metric name might be enough, but there might be duplicates of labels too - hard to debug without extra entry.

We can add in next PR though, thanks!